x-pack/plugins/observability_solution/synthetics/public/apps/synthetics/utils/validators/is_url_valid.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+const allowedProtocols = ['http:', 'https:'];
+
+export const isUrlValid = (url?: string | null) => {
+  if (!url) return false;
+
+  try {
+    const urlParsed = new URL(url);
+
+    if (
+      allowedProtocols.includes(urlParsed.protocol) &&
+      url.startsWith(`${urlParsed.protocol}//`)
+    ) {
+      return true;
+    }
+
+    return false;
```

Comment:
nit 
```suggestion
   return allowedProtocols.includes(urlParsed.protocol) && url.startsWith(`${urlParsed.protocol}//`)
```

Reply:
thanks for the suggestion,  much cleaner. The tests should pass after the update